Github Issue Reader/ViewController.swift

```diff
@@ -130,4 +156,52 @@ class ViewController: UIViewController {
             }
         }
     }
+      
+      // while it's loading here is a cute UI thing to tell you the networking call is happening from "now" up to 5 seconds after launching the call which will make it time out and quits/crashes
```

Comment:
This chunk was previously removed in favor of loading happening in the LoadingCell, it laying here causes ViewController to fail to compile as well